buildSrc/src/main/kotlin/ReleaseScripts.kt

```diff
@@ -56,11 +56,11 @@ private fun Semver.getGitTag(): String = if (build.isNotEmpty()) build.joinToStr
 data class KotlinVersion(val javaVersion: Semver, val kotlinMinor: Int, val isSnapshot: Boolean) {
     companion object {
         fun fromVersionString(versionString: String): KotlinVersion {
-            val versionStringSegments = """^(\d+\.\d+\.\d+)\.(\d+)(-.*\+\w+)?(-SNAPSHOT)?$"""
+            val versionStringSegments = """^(\d+\.\d+\.\d+)\.(\d+)(-.*?)??(-SNAPSHOT)?$"""
```

Comment:
Docker started using versions like [`4.6.0-alpha.0`](https://mvnrepository.com/artifact/com.pulumi/docker/4.6.0-alpha.0), which doesn't parse with the old regex.